### PR TITLE
Modifications related to deployment

### PR DESCRIPTION
**Pull request by fredy20501**
- Removed unnecessary files & comments
- Fixed the configs to work locally
- Added code to serve frontend as static files once they are built

### CODE REVIEW

**Comment by fredy20501** [comment]
@JoaoPontes98  @PEKennedy  @ihatecsv  
Since the pull request for the deployment_mods got merged quickly  I didn't have the chance to comment/review it. The changes that were merged prevent us from running the app locally so I fixed it in this pr. 

Also, note that the frontend should be built using `npm run build` which builds the frontend into a set of static files in a directory called `dist`. I added code that I think makes it work with the built files from the `dist` folder.

Could you guys review the changes and check if they work on the server? (follow the steps in the readme) 
Thanks!

**Comment by fredy20501** [comment]
Also would it be possible to get the app running on the server constantly? That way Dawn will be able to try it out on her own time and give us feedback.

**Review by ravenmills3 — APPROVED** [review]

**Comment by PEKennedy** [comment]
This works fine both locally and on google cloud.
Though I put the frontend production port into a new config file and changed it to 3000.

File: README.md [hunk 1]
```diff
@@ -89,24 +89,25 @@ npx npm-merge-driver install --global
...
-- Then to kill the process: run 
-ps ux
+nohup npm run production
 ```
-Look for the 2 processes labelled 'node home/.../vue-cli-...' and 'node home/.../nodemon server/index.js', get their IDs.
-- Run '$Kill -9 ID' for both of them
+- `nohup` tells ubuntu not to kill the process when we log out
+- Then to kill the process run `ps ux`
+- Look for the 2 processes labelled `node home/.../vue-cli-...` and `node home/.../nodemon server/index.js`, get their IDs
+- Run `$Kill -9 ID` for both of them
```
Comment (PEKennedy) [inline README.md:110]: Outdated info, we should talk about using tmux to host as seen here: https://askubuntu.com/questions/8653/how-to-keep-processes-running-after-ending-ssh-session

File: src/production.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const app = express();
+app.use(cors())
+
+// Setup the frontend if in production (source: https://dennisreimann.de/articles/vue-cli-serve-express.html)
+// Since the frontend is only a set of static pages once built we can serve them as static files on a server
+const publicPath = resolve(__dirname, '../dist')
+const staticConf = { maxAge: '1y', etag: false }
+app.use(express.static(publicPath, staticConf))
+app.use('/', history())
+var config = require('./config.js');
```
Comment (PEKennedy) [inline src/production.js:15]: Forgot to add the TravisCI env check like the backend has, if there is a CI issue bc of this, we should add the env check here

**Review by PEKennedy — APPROVED** [review]
Some small comments on stuff we probably want in the future. But the changes as they are, are good changes.

File: src/production.js (hunk 2 above)
Comment (fredy20501) [reply src/production.js:15]: There won't be any issues since TravisCI doesn't run `npm run production`

**Pull request merged**